ims/einstein/ceph.py

```diff
@@ -84,9 +84,10 @@ def list_images(self):
         return self.rbd.list(self.context)
 
     @log
-    def create_image(self, img_id, img_size):
+    def create_image(self, img_id, img_size, old_format=True, features=0):
```

Comment:
also, we are changing a function in Einstein, did you test it to make sure that it doesn't break the current operations.